### ISSUE TITLE
Cli - When any action fails , errors are not presented to the user.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11491]
rancher cli version - v1.0.0-alpha5

When any action fails , errors are not presented to the user.

In my case , I logged in as user who does not have access to any clusters ( so no default cluster/project selected).
Created a cluster.
Then tried to add a project , which actually fails which is expected.
But user is not presented with any error messages in this case.
```
./rancher project create hello
./rancher project ls
ID        NAME      STATE     DESCRIPTION
```

**Comment by deniseschannon** [comment]
Available with v1.0.0-alpha7

**Comment by sangeethah** [comment]
Tested with v1.0.0-alpha7

When cli commands fail, errors are presented to the user.
```
Sangeethas-MBP:rancher-v1.0.0-alpha7 sangeethahariharan1$ ./rancher project create project1
WARN[0001] No default project set, run `rancher login` again. Some commands will not work until project is set 
FATA[0004] Bad response statusCode [422]. Status [422 Unprocessable Entity]. Body: [fieldName=clusterId, message=, baseType=error, code=MissingRequired] from [https://18.219.235.13/v3/projects] 
```
